edx_sga/sga.py

```diff
@@ -294,7 +295,8 @@ def student_state(self):
             "annotated": annotated,
             "graded": graded,
             "max_score": self.max_score(),
-            "upload_allowed": self.upload_allowed(submission_data=submission)
+            "upload_allowed": self.upload_allowed(submission_data=submission),
+            "solution": force_text(self.solution) if self.answer_available() else '',
```

Comment:
Instinctively I would think we should eliminate the `solution` key from the `student_state` return value, but it's not important enough to hold up the PR